hack/patch-converter-runner/README.md

```diff
@@ -0,0 +1,46 @@
+# Patch Converter Runner 
+
+## Overview
+
+The `patch-converter-runner` program invokes converter library to generate shoot from runtime resources. The purpose of this application is to allow developers to:
+- quickly reproduce problems with updating existing Gardener clusters
+- debug converter without a need to run the whole KIM system
+
+## Building the Binary
+
+To build the `patch-converter-runner` binary navigate to the `hack` folder, and execute the following command:
+
+```sh
+  go build -o ./bin/patch-converter-runner ./patch-converter-runner/cmd/main.go
+```
+
+## Usage
+
+### Options
+
+- `--runtime-path`: The path to the runtime CR file
+- `--shoot-path`: The path to the existing shoot CR file
+- `--kubeconfig-path`: The path to the KCP kubeconfig file
+- `--outputPath`: The path to the output shoot CR file
+
+### Example
+
+To run the `patch-converter-runner` program, execute the following command:
+```sh
+patch-converter-runner -runtime-path=<runtime CR fie path> -shoot-path=<shoot CR path> -kubeconfig-path=<KCP kubeconfig path> -outputPath=<output shoot CR path>
```

Comment:
Maybe the `outputPath` is not necessary and can be just replaced with usage for the current directory

Reply:
I decided to leave it as it is